raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2639,15 +2687,19 @@ private long maybeSendBeginQuorumEpochRequests(
                                 channel.listenerName()
                             )
                         )
-                    ),
+                    );
+
+            timeUntilNextBeginQuorumSend = maybeSendRequest(
+                currentTimeMs,
+                voters
+                    .voterKeys()
+                    .stream()
+                    .filter(key -> key.id() != quorum.localIdOrThrow())
+                    .collect(Collectors.toSet()),
+                nodeSupplier,
                 this::buildBeginQuorumEpochRequest
             );
             state.resetBeginQuorumEpochTimer(currentTimeMs);
-            logger.trace(
```

Comment:
what's the rationale for dropping the `trace` message here?

Reply:
It didn't find it useful. Probably more useful to just enable RPC logging instead.